nix/pkgs/core.nix

```
@@ -18,13 +18,13 @@ rec {
 
     # alas, we cannot specify --features in the virtual workspace
     # run the specific slow tests in the holochain crate
-    cargo test --no-run --all-targets --manifest-path=crates/holochain/Cargo.toml --features slow_tests,test_utils,build_wasms,db-encryption -- --nocapture --test-threads 1
+    cargo test --no-run --all-features --all-targets --manifest-path=crates/holochain/Cargo.toml --features slow_tests,test_utils,build_wasms,db-encryption
```

confusing to specify a list of features and also all features...

```suggestion
    cargo test --no-run --all-features --all-targets --manifest-path=crates/holochain/Cargo.toml
```